### DISCUSSION TITLE
What is the PHP version and enabled extensions?

### DISCUSSION THREAD

**Discussion by webNeat** [geeksblabla/blanat#126] [Q&A] [answered]
Hello,

I would like to know the output of the following commands

```
php -v
```

```
php -m
```

Thanks

**Comment by aboullaite** [comment, marked as answer]
```
$ php -v
PHP 8.2.7 (cli) (built: Jun  9 2023 19:37:27) (NTS)
```
$ php -m
[PHP Modules]
calendar
Core
ctype
date
exif
FFI
fileinfo
filter
ftp
gettext
hash
iconv
json
libxml
openssl
pcntl
pcre
PDO
Phar
posix
random
readline
Reflection
session
shmop
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
Zend OPcache
zlib

[Zend Modules]
Zend OPcache
```

**Reply by webNeat** [reply]
Thanks!
